k8s/elasticsearch/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,89 @@
 actions:
-- name: dummy test
+- name: Wait until all expected nodes are connected to the cluster
   bashTest:
     script: |-
-      exit 0
+      function get_nodes_count() {
+        curl -s "${HEALTH_URL}" | jq '.number_of_nodes'
+      }
+
+      expected_nodes="${REPLICAS}"
+
+      timeout=300
+      timeout_time=$(( $(date +%s) + timeout ))
+
+      # Wait for all nodes to join the cluster
+      nodes_count="$(get_nodes_count)"
+      while [[ "${nodes_count}" -ne "${expected_nodes}" ]] \
```

Comment:
Just a question: Does it mean that running Pod can be still not connected to the cluster?

Reply:
Yes.